design/hardware-status.md

```diff
@@ -0,0 +1,159 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# hardware-status
+
+## Status
+
+provisional
+
+## Table of Contents
+
+   * [Title](#title)
```

Comment:
It looks like the inline stuff for gh-md-toc is gone and this may be out of date. We can fix that with a follow-up patch.